src/facebook/facebookdatatypesyncadaptor.cpp

```diff
@@ -220,7 +220,7 @@ void FacebookDataTypeSyncAdaptor::signOnError(const SignOn::Error &error)
 
     // if the error is because credentials have expired, we
     // set the CredentialsNeedUpdate key.
-    if (error.type() == SignOn::AuthSession::UserInteractionError) {
+    if (error.type() == SignOn::Error::UserInteraction) {
         setCredentialsNeedUpdate(account);
```

Comment:
Yes, I did a symbol search on the global grok thing, and only sailfish-components-accounts and sociald used it - so hopefully it's just those two places.